cartography/models/aws/ec2/network_acl_rules.py

```diff
@@ -21,6 +21,7 @@ class EC2NetworkAclRuleNodeProperties(CartographyNodeProperties):
     fromport: PropertyRef = PropertyRef('FromPort')
     toport: PropertyRef = PropertyRef('ToPort')
     cidrblock: PropertyRef = PropertyRef('CidrBlock')
+    Ipv6CidrBlock: PropertyRef = PropertyRef('Ipv6CidrBlock')
     egress: PropertyRef = PropertyRef('Egress')
```

Comment:
@cmm-lyft  a bit late to comment but probably more consistent to use lowercase `ip`. If we haven't cut a release, maybe can still do a quick fix?

Reply:
oh yeah good catch. lol sorry i got too eager with the approval